### PR TITLE
Microsoft Defender Endpoint Log Parser for Linux, version 1.2

### PR DESCRIPTION
**Pull request by bmrodrigues**
New Menu, new help, code and performance optimization, new complied version and new README.MD

### CODE REVIEW

File: linux/LinuxMDEparser/README.MD [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# LinuxMDEparser
+Microsoft Defender Endpoint Log Parser for Linux.
+
+# About
+The LinuxMDEparser is a Windows tool that allows you to export or parse MDE for Linux log files to Excel, making it easier to filter and search the logs. After collecting the MDE logs from any Linux machine, you can then run this tool on your on Windows.
```
Comment (maximvelichko) [inline linux/LinuxMDEparser/README.MD:5]: Please remove references to Windows as the only supported platform, as from what I can see nothing in your code requires Windows. This repository is for cross-platform Defender. Mandating Windows for handling non-Windows product raises too many questions/do not look right for customers who use no Windows in their daily work.

It is totally OK to include instructions on how to run it on Windows, complementing instructions on how to run the tool from bare Python.

File: linux/LinuxMDEparser/README.MD [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+# Usage
+LinuxMDEparser {Command} <option>
+
+EX: LinuxMDEparser.exe real-time-protection --convert
```
Comment (maximvelichko) [inline linux/LinuxMDEparser/README.MD:10]: .exe is for Windows only, please remove to make it compatible with all supported platforms

**Review by maximvelichko — CHANGES_REQUESTED** [review]

File: linux/LinuxMDEparser/README.MD (hunk 1 above)
Comment (bmrodrigues) [reply linux/LinuxMDEparser/README.MD]: Committed as requested

File: linux/LinuxMDEparser/README.MD (hunk 2 above)
Comment (bmrodrigues) [reply linux/LinuxMDEparser/README.MD]: Moved the .exe references to the README.MD inside the Dist forlder

File: linux/LinuxMDEparser/README.MD [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+Files are included when you collect the Linux - "mdatp diagnostic create" command in Linux, except for real_time_protection.json - this ones need to be collected locally in a terminal window:
+
+mdatp diagnostic real-time-protection-statistics --output json > real_time_protection.json
+
+More details https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/linux-support-perf
+
+# Help
+You can always check the available options by running: LinuxMDEparser -h
+
+usage: LinuxMDEparser.exe [command] [--option]
```
Comment (maximvelichko) [inline linux/LinuxMDEparser/README.MD:24]: Still .exe

**Review by maximvelichko — APPROVED** [review]

File: linux/LinuxMDEparser/README.MD (hunk 3 above)
Comment (bmrodrigues) [reply linux/LinuxMDEparser/README.MD:24]: This is the help from the python and I rather prefer leave it like this. I can remove the help output if you prefer ...

Comment (maximvelichko) [reply linux/LinuxMDEparser/README.MD:24]: @bmrodrigues , the help output will be without .exe on all platforms but Windows (and I believe that macOS/Linux is our primary platform for this repository). I know that many non-Windows users will get confused (and will open issues agains our code) if they see .exe and do not know where to get it. On the other hands Windows users will most probably ignore a missed .exe - as it is common to run executables on Windows omitting .exe

**Review by bmrodrigues — COMMENTED** [review]
.exe should now be removed from README.MD

**Pull request merged**